XenAdmin/Controls/HaNtolControl.cs

```diff
@@ -237,7 +239,6 @@ private void updateNtol()
                         });
                 }
 
-                waitingNtolUpdate.WaitOne();
                 log.Debug("Thread woken");

```

Comment:
I haven't looked at the behaviour, but i think this should be kept adjacent to the moved line.

Reply:
Oops you're right, will fix :)